<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskInfo</class>
 <widget class="QDialog" name="AskInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>195</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="confirmLayout">
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="askInfoLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Your name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="text">
       <string>Player</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="chooseLevelLabel">
      <property name="text">
       <string>Choose level</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="levelSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="levelSpinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>levelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>levelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>levelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
